src/ontology/edits/oeo-model.omn

```diff
@@ -3079,7 +3079,7 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1711",
 Class: OEO_00360020
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A region of relevance is a spatial region that is used in a study or analysis."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A region of relevance is a spatial region that is used in a study or analysis, or is part of spatial planning."@en,
```

Comment:
Since changed, region of relevance should also appear in changelog and get new term trackers.